pkg/meta/version.go

```diff
@@ -2,7 +2,7 @@ package meta
 
 const (
 	// CLIAPIVersion used to communicate with user e.g. longhorn-manager
-	CLIAPIVersion    = 5
+	CLIAPIVersion    = 6
```

Comment:
note: This is not be effective after 1.3, because the proxy has been introduced. However, it's still needed for releases < 1.3.

Reply:
Still bump the version because some test cases still rely on the version number.

Reply:
Yeah, I see, as discussed here, https://github.com/longhorn/longhorn-engine/pull/740#discussion_r963031031.